dali/python/nvidia/dali/data_node.py

```diff
@@ -189,25 +190,29 @@ def process_index(idx, dim):
                 return False
 
         if not isinstance(val, tuple):
-            val = (val,)
+            val = (val, )
         d = 0
         for v in val:
             if process_index(v, d):
                 d += 1
 
         if len(new_axis_names) != 0:
             if len(new_axis_names) != len(new_axes):
-                raise ValueError("New axis name must be specified for all axes or none.");
+                raise ValueError("New axis name must be specified for all axes or none.")
             new_axis_names = "".join(new_axis_names)
         else:
             new_axis_names = None
 
         slice_args = {}
         for i, (at, lo, hi, step) in enumerate(idxs):
-            if at   is not None: slice_args["at_%i" % i] = at
-            if lo   is not None: slice_args["lo_%i" % i] = lo
-            if hi   is not None: slice_args["hi_%i" % i] = hi
-            if step is not None: slice_args["step_%i" % i] = step
+            if at is not None:
+                slice_args["at_%i" % i] = at
+            if lo is not None:
+                slice_args["lo_%i" % i] = lo
+            if hi is not None:
+                slice_args["hi_%i" % i] = hi
+            if step is not None:
+                slice_args["step_%i" % i] = step
```

Comment:
I find the original formatting easier to read.

Reply:
I asked around and new formatting got positive feedback, so since it is a matter of personal preference I'm going to leave it as it is.